### PR TITLE
6-java-project-configuration

### PR DESCRIPTION
**Pull request by Mateusz-Dobrzynski**
[6] create a default project via Maven

Check if you can run this project on your machines.
@hubertborysowski, request changes according to code guidelines once you're done with them.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+insert_final_newline = true
+trim_trailing_whitespace = true
+max_line_length = 80
+
+[*.sh]
+end_of_line = lf
+
+[*.java]
+indent_size = 4
+max_line_length = 120
```
Comment (hubertborysowski) [inline .editorconfig:17]: According to Google style max_line_length should be 100 (§4.4 Column limit: 100).
There is also a problem when it comes to exceptions such as package instructions, import instructions, etc., which should not have any column limit (§3.2, §3.3.2). 
If you are sure that this extension will allow such exceptions it is only a matter of changing the number from 120 to 100.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+          <skip>${skipTests}</skip>
+        </configuration>
+        <executions>
+          <execution>
+            <id>checkstyle</id>
+            <phase>validate</phase>
+            <goals>
+              <goal>check</goal>
+            </goals>
+          </execution>
```
Comment (hubertborysowski) [inline pom.xml:87]: Everything seems to work in general, but I get these warnings after compiling and running the program. I cannot find a way to fix it as for now.
2023-06-28 19_24_20-pom xml - meals-management-system-6-java-project-configuration - Visual Studio C

**Review by hubertborysowski — CHANGES_REQUESTED** [review]

**Comment by Mateusz-Dobrzynski** [comment]
Since this configuration does not include Springboot, I'm closing this PR

**Comment by Mateusz-Dobrzynski** [comment]
Since this configuration does not include Springboot, I'm closing this PR

**Pull request closed without merging**